.gitignore

```
@@ -11,6 +11,8 @@ Thumbs.db
 *.log*
 node_modules/
 
+upload
```

```suggestion
upload/
```
для консистентности с другими паттернами